src/main/java/lotto/domain/WinningRank.java

```diff
@@ -8,38 +8,49 @@
 public enum WinningRank {
 
     FIRST(6, 2_000_000_000),
-    SECOND(5, 1_500_000),
-    THIRD(4, 50_000),
-    FOURTH(3, 5_000),
+
+    SECOND(5, 30_000_000),
```

Comment:
개행이 없는 것이 더 가독성이 좋을 것 같습니다.
```suggestion
    SECOND(5, 30_000_000),
```

Reply:
의도한 개행은 아니였지만 ㅎㅎ 꼼꼼한 리뷰 감사합니다 😄